pkgs/development/python-modules/tensorrt/default.nix

```diff
@@ -0,0 +1,58 @@
+{ lib
+, python
+, buildPythonPackage
+, autoPatchelfHook
+, unzip
+, cudaPackages
+, cudatoolkit
```

Comment:
```diff
- , cudatoolkit
```

Cf. comment about `cudaPackages` above.
The convention is to `inherit (cudaPackages) cudatoolkit;` instead